pkg/config/template.go

```diff
@@ -138,6 +138,9 @@ type configuration struct {
 		Local                  *struct {
 			Path string
 		}
+		Mem *struct {
+			ReuseStore bool `mapstructure:"reuse_store"`
```

Comment:
It's an in-memory test only implementation, I don't understand why we would like to add this flag to the configuration of lakeFS